docs/source/conf.py

```diff
@@ -30,7 +30,9 @@
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = [
-    "sphinx_rtd_theme", "sphinx.ext.intersphinx"
+    "sphinx_rtd_theme",
+    "sphinx.ext.intersphinx",
+    "pydoctor.sphinx_ext.help_output",
```

Comment:
Feel free to suggest a better name.

Note that we already have pydoctor.sphinx which is the code to generate the Sphinx inventory.